webapp/models.py

```diff
@@ -0,0 +1,41 @@
+from sqlalchemy import Column, Date, Integer, String, DateTime
+from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy.sql import func
+
+class Base(DeclarativeBase):
+  pass
+
+
+class PreviousSession(Base):
+  __tablename__ = "previous_sessions"
+
+  id = Column(Integer, primary_key=True)
+  topic = Column(String, nullable=False)
+  owner = Column(String, nullable=False)
```

Comment:
Just a general comment, not sure if we have created any indexes for table columns that's used for search queries? You can do this either when creating the tables or add indexes to columns afterwards. I assume that masterclasses is a more read heavy app so speeding up read queries might be good. However, If we never do conditional queries then not having indexes is also fine

Reply:
Currently we do search on the frontend and maybe after we're sure this PR works well, we can work on performance improvements.